.github/workflows/CI.yml

```diff
@@ -881,6 +884,61 @@ jobs:
         run: |
           cd testing-framework/terraform/eks && terraform destroy -auto-approve
 
+  e2etest-eks-fargate:
+    runs-on: ubuntu-latest
+    needs: [get-testing-suites, e2etest-release, e2etest-preparation]
+    strategy:
+      fail-fast: false
+      max-parallel: 1
+      matrix: ${{ fromJson(needs.get-testing-suites.outputs.eks-fargate-matrix) }}
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Cache if success
+        id: e2etest-eks-fargate
+        uses: actions/cache@v2
+        with:
+          path: |
+            VERSION
+          key: e2etest-eks-fargate-${{ github.run_id }}-${{ matrix.testcase }}
```

Comment:
IMO github.run_id is unique enough to cover the key.

Reply:
This is copied from the eks tests. Just changed the key name to e2etest-eks-fargate from e2etest-eks